src/components/NavBar/NavBarWithoutSearch/NavBarWithoutSearch.tsx

```diff
@@ -28,8 +31,26 @@ const NavBar: React.FC<{
     }
   };
 
+  const onHoverHamburger = () => {
+    if (isMobile) {
+      return;
+    } else {
+      if (!menuOpen) {
+        setMenuOpen(true);
+        trackMobileMenuOpen();
+      }
+    }
+  };
+
   const closeMenu = () => setMenuOpen(false);
 
+  const onMouseLeave = (e: React.MouseEvent<{}>) => {
+    // Do not close when the user hovers on the top bar (including the button to close the menu)
+    if (e.clientY > 64) {
+      closeMenu();
```

Comment:
Either I don't understand the intent here or I think it's not working...  If I hover over the hamburger to open the menu and then move the cursor down below the 64px navbar line and then back up to the navbar, then the menu closes.  Normally people won't do this.  They'll just drag their cursor down into the menu and click something...  but I could imagine that somebody who is maybe not super coordinated might slide their mouse around a bit on that journey and accidentally go below / above that magic 64px line on their journey to trying to click something.

What's the intended behavior?

Reply:
oooo strange. yeah its being a little buggy. the intended behavior is that the close-menu-onMouseLeave doesn't apply to the 64px navbar. this works when moving from the hamburger to the navbar, but it looks like it doesnt work if you first hover on the menu before moving back up to the navbar.

Reply:
oh! fixed it. 64 needed to be increased to 84, since the new navbar is 20px larger (to make space for the search bar)